.github/workflows/linter.yml

```diff
@@ -58,3 +58,4 @@ jobs:
           # Change to 'master' if your main branch differs
           DEFAULT_BRANCH: master
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FILTER_REGEX_EXCLUDE: "*/usr/bin/hna.lua*"
```

Comment:
`*/usr/bin/hna.lua*` => `.*/usr/bin/hna\.lua.*`

Reply:
resolved in current version. I ommited the one `\` though. codium complained, that the backslash would result in invalid YAML-Syntax.

Reply:
In that case you could write it as `[.]` or try using double escaping `\\.` as otherwise the `.` matches any single character except for line terminators.

Reply:
corrected again. :) Lets see, if the CI takes this.